### ISSUE TITLE
Full Multi Camera Support for Android 

### ISSUE THREAD

**Issue by DarkMatter-999** [j20001970/GDMP#46]
How can we support multiple cameras that is ultrawide and zoom using the current implementation of Android's `GDMPCameraHelper`

In case of Windows and Linux hosts since it uses OpenCV by specifying the index of camera we are able to get every camera, how about doing that on Android hosts.

We can have a function similar to this and instead of only FRONT and BACK we will have cameras based on their indexes.
```java
private CameraSelector getCameraSelectorByIndex(int index) {
        ListenableFuture<ProcessCameraProvider> cameraProviderFuture = ProcessCameraProvider.getInstance(activity);
        try {
            ProcessCameraProvider cameraProvider = cameraProviderFuture.get();
            List<CameraInfo> availableCameras = new ArrayList<>(cameraProvider.getAvailableCameraInfos());

            if (index < 0 || index >= availableCameras.size()) {
                throw new IndexOutOfBoundsException("Camera index out of bounds.");
            }

            CameraInfo selectedCameraInfo = availableCameras.get(index);

            return new CameraSelector.Builder().selectCamera(selectedCameraInfo).build();

        } catch (InterruptedException | ExecutionException e) {
            Log.e(TAG, "Error selecting camera by index: " + e.getMessage());
            return null;
        }
    }
```

PS - I am not able to get my build for Android to succeed following the instructions in the `BUILDING.md`.
Python - 3.12
JDK - 11/17/22
AndroidSDK - 34.0.0
AndroidNDK - 21.0.xx...

**Comment by DarkMatter-999** [comment]
Yeah this one seems better as we can first handle the `DEFAULT_FRONT_CAMERA` and `DEFAULT_BACK_CAMERA` as the standard and if other cameras exist then choose them.

**Comment by j20001970** [comment]
Hi, since GDMP v0.5, camera helper has been considered deprecated, and no new features will be developed for it. You might want to consider CameraServer-based methods for accessing camera frames on Godot Engine in future.

I will close the issue as `MediaPipeCameraHelper` and `GDMPCameraHelper` for Android's backend is no longer developed, please open the issue on [CameraServerExtension](https://github.com/j20001970/godot-cameraserver-extension) if accessing multiple cameras on Android is still a problem for you.
